WhisperAPI/WhisperAPI/Controllers/SuggestionsController.cs

```diff
@@ -13,10 +15,18 @@ public SuggestionsController(ISuggestionsService suggestionsService)
             this._suggestionsService = suggestionsService;
         }
 
-        [HttpGet]
-        public IActionResult GetSuggestions()
+        [HttpPost]
+        [Route("")]
```

Comment:
Je crois que cette ligne sert à rien, par défaut c'est déjà ça

Reply:
C'était juste pour rendre plus explicite, au même titre mettre "this." qui ne sert pas plus a grand chose